Assets/_BForBoss/_LevelDesign/Scripts/LevelDesignFeedbackCaller.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using UnityEngine;
+
+namespace Perigon.Utility
+{
+    public class LevelDesignFeedbackCaller : MonoBehaviour
+    { 
+#if UNITY_EDITOR
+        private void OnGUI()
+        {
+            Event evt = Event.current;
+            if (evt.isKey && evt.keyCode == LevelDesignFeedbackWindowListener.WindowKeyCode)
```

Comment:
I'm pressing the windows key and its not opening anything. not sure how this is activated

Reply:
Oh do Mac not work with pressing the F10 key?

Also to make sure, you are on scene with the `Tools Menu - Editor Only` and are in play mode, in the editor?

Reply:
I was on Windows. pressing the `WindowsKeyCode`. Where dose it decide that it's F10?

Reply:
Imo, the keycode should be in this class, bcecause I feel like it's the job for 
`LevelDesignFeedbackWindowListener` to `OpenLebelDesignFeedbackWindow`, but not so much determine which key has to be pressed to open the window

Reply:
So for example, if on Mac it should be Ctrl + A, it's the View/MonoBehaviour that determines the keycode. 

And the `LevelDesignFeedbackWindowListener` is focused on opening.

Reply:
The way we use listeners at work is that the Listener class decides what it is listening for. It will seem weird to open a listener class and be unsure what the Method is listening for, in my opinion. The View should just be concerned if it is pressing the correct command the listener will know to act.

We can discuss this next time we meet, or when discussing the second iteration!

Reply:
So listener classes are normally interfaces you implement: 
https://stackoverflow.com/questions/7510532/what-is-the-purpose-of-a-listener-class-in-a-large-project

And currently your listener class also opens up a window which are 2 separate jobs. 
So you can setup a listener interface for your listener class to implement (and name it something else like `LevelDesignFeedbackPresenter` or something